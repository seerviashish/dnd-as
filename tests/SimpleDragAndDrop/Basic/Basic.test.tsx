import renderer from 'react-test-renderer';
import { expect, it } from 'vitest';
import { Basic } from '../../../src/examples/SimpleDragAndDrop';

it('SimpleDragAndDrop/Basic tests', () => {
  const tree = renderer.create(<Basic />).toJSON();
  expect(tree).toMatchSnapshot();
});
