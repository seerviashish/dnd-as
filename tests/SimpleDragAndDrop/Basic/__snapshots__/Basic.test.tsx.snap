// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`SimpleDragAndDrop/Basic tests 1`] = `
<div
  className="MuiBox-root css-0"
>
  <div
    className="MuiBox-root css-0"
  >
    Draggable Element
  </div>
  <div
    className="MuiBox-root css-0"
  >
    Drop Area
  </div>
</div>
`;
